pht_federated/client/storage/fs.py

```diff
@@ -7,7 +7,7 @@
 
 class MinioBuckets(enum.Enum):
     MODELS = 'models'
-    DATASETS = 'datasets'
+    DATASETS = 'discoveries'
```

Comment:
datasets = discoveries? :D why is this needed?

Reply:
hab ich nie etwas geschrieben in dem file soweit ich weiß

Reply:
steht bei mir auch dass du das als letzte Person verändert hast :D